crates/uv-cli/src/lib.rs

```diff
@@ -2753,6 +2770,62 @@ pub struct TreeArgs {
     pub python: Option<String>,
 }
 
+#[derive(Args)]
+#[allow(clippy::struct_excessive_bools)]
+pub struct ExportArgs {
+    /// Export the dependencies for a specific package in the workspace.
+    ///
+    /// If not in a workspace, or if the workspace member does not exist, uv
+    /// will exit with an error.
+    #[arg(long)]
+    pub package: Option<PackageName>,
+
+    /// The format to which `uv.lock` should be exported.
+    ///
+    /// At present, only `requirements.txt` is supported.
+    #[arg(long)]
+    pub format: ExportFormat,
```

Comment:
Right now this is a required argument, even though we only support one format. We could make it optional, or even remove it? But if we add formats in the future, it will then be breaking.

Reply:
I don't mind this, kind of funny though. What else might we support?

Reply:
Actually, we can just make this an optional with a default.